dix/req.h

```diff
@@ -253,6 +253,8 @@ struct m0_dix_req {
 	 * starting key in DIX_NEXT request.
 	 */
 	uint32_t                     *dr_recs_nr;
+	/** Minimum successful CAS operations. */
```

Comment:
Suggestion:
```
/**
  * Minimum number of successful CAS operations to treat
  * parent DIX operation as successful.
  * /
```
Also in a comments we don't use word "replicas", using this field we are comparing number of successful results, so may be we can use _dr_min_success_ instead of _dr_min_replicas_? That would be more descriptive, I think.

Reply:
That's fine, I renamed everything to "success"